src/Sylius/Bundle/AttributeBundle/DependencyInjection/Compiler/RegisterAttributeFactoryPass.php

```diff
@@ -30,13 +34,29 @@ public function process(ContainerBuilder $container)
             return;
         }
 
-        $registry = $container->getDefinition('sylius.registry.attribute_type');
+        $baseFactoryDefinition = new Definition(
+            Factory::class,
+            array(
+                new Parameter('sylius.model.product_attribute.class'),
```

Comment:
Attribute bundle should not know about ``product_attribute``, we have a bad coupling bug in the whole AttributeBundle...